server/src/main/java/org/jboss/as/server/Main.java

```diff
@@ -366,6 +367,12 @@ public static ServerEnvironmentWrapper determineEnvironment(String[] args, Prope
                     } else {
                         gitBranch = arg.substring(idx + 1);
                     }
+                } else if (arg.startsWith(CommandLineConstants.STABILITY)) {
```

Comment:
For now, if ProductConfig.getDefaultStability() == DEFAULT this should fail. A similar check can be done in CommandLineArgumentUsageImpl to guard allowing the argument.

Reply:
Fixed.  Also removed from command line usage, when stability has only 1 possible value.